docs/dev/src/antora.yml

```diff
@@ -3,3 +3,4 @@ version: master
 
 nav:
 - modules/java/nav.adoc
+- modules/spring/nav.adoc
```

Comment:
We might want to not add this for now, otherwise it might be released with a new version of the sdk, right?